app/Http/Controllers/AnswerController.php

```diff
@@ -4,12 +4,26 @@
 
 use App\Quiz;
 use App\Answer;
+use App\Lesson;
 use App\Activity;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use App\User;
 
 class AnswerController extends Controller
 {
+    public function index()
+    {
+    }
+
+    public function show($id)
+    {
+        $user = User::find($id)->first();
+        $answers = $user->getLearnedWords($id);
```

Comment:
I think when you put `User` in `show()` function it will automatically find the user e.g
```
public function show(User $id) {
...
}
```

Reply:
resolved in  58ae3fd